src/main/java/io/workflow/bpmnsimulator/fieldvalidator/AssigneeFieldValidator.java

```diff
@@ -0,0 +1,29 @@
+package io.workflow.bpmnsimulator.fieldvalidator;
+
+import io.workflow.bpmnsimulator.model.Field;
+import io.workflow.bpmnsimulator.model.Step;
+import lombok.extern.slf4j.Slf4j;
+import org.camunda.bpm.engine.task.Task;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.Nonnull;
+import java.util.Objects;
+
+@Slf4j
+@Service
+class AssigneeFieldValidator implements FieldValidator {
+
+    @Override
+    public FieldValidationResult validate(@Nonnull final Step step, @Nonnull final Task task) {
+        final boolean isValid = Objects.equals(task.getAssignee(), step.getAssignee());
```

Comment:
assignee can be null
candidate user and candidate group must be check

Reply:
It works fine with `null` assignee. But I've created a ticket to add a test for that: https://github.com/mohsen-ebrahimi/bpmn-simulator/issues/6

Also I've added two other tickets to support `candidateUser` and `candidateGroup`:
https://github.com/mohsen-ebrahimi/bpmn-simulator/issues/4
https://github.com/mohsen-ebrahimi/bpmn-simulator/issues/5